modules/axhal/src/platform/aarch64_bsta1000b/mod.rs

```diff
@@ -37,6 +37,7 @@ pub(crate) unsafe extern "C" fn rust_entry(cpu_id: usize, dtb: usize) {
 
 #[cfg(feature = "smp")]
 pub(crate) unsafe extern "C" fn rust_entry_secondary(cpu_id: usize) {
+    let cpu_id = cpu_id / 256;
```

Comment:
do not hard code this
you can refer to https://github.com/arceos-hypervisor/arceos/blob/c62b386e829cc2252885b90b9d5d7aad9ead6a64/modules/axhal/src/platform/aarch64_bsta1000b/mp.rs#L6C5-L6C11

Reply:
yeah, this is appropriate